### PR TITLE
CAD-2477:  clean up metric name space

### PR DESCRIPTION
**Pull request by deepfire**
1. Consolidate all metrics under `cardano.node.metrics`.
2. Update configs.
3. Eliminate mentions of stale namespace references.

Now:
Screenshot from 2021-01-15 19-27-04

Before:
Screenshot from 2021-01-15 01-25-50

### CODE REVIEW

File: configuration/defaults/byron-staging/configuration.yaml [hunk 1]
```diff
@@ -224,8 +224,6 @@ options:
       - EKGViewBK  
     # Uncomment it to send 'cardano.node.metrics' to 'TraceForwarderBK' as well.
     # - TraceForwarderBK
-    cardano.node-metrics:
-      - EKGViewBK
```
Comment (erikd) [inline configuration/defaults/byron-staging/configuration.yaml:228]: Is there some mechanism to make sure these get fixed in the config tempate devops uses?

Comment (deepfire) [reply configuration/defaults/byron-staging/configuration.yaml:228]: I've made @jbgi aware of this, yes.

Comment (erikd) [reply configuration/defaults/byron-staging/configuration.yaml:228]: So this is currently a manual process? Can it not be automated in some way to make sure it always gets done?

Comment (deepfire) [reply configuration/defaults/byron-staging/configuration.yaml:228]: This template is in a different repository -- `iohk-nix`, and it contains defaults for multiple services.

We've actually been discussing this topic a while ago, but it wasn't feasible then.

cc @disassembler

**Review by denisshevchenko — APPROVED** [review]
LGTM

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 2]
```diff
@@ -353,12 +354,13 @@ teeTraceChainTip
   => TraceOptions
   -> MVar (Maybe (WithSeverity (ChainDB.TraceEvent blk)), Integer)
   -> Trace IO Text
+  -> Trace IO Text
   -> Tracer IO (WithSeverity (ChainDB.TraceEvent blk))
-teeTraceChainTip TracingOff _ _ = nullTracer
-teeTraceChainTip (TracingOn trSel) elided tr =
+teeTraceChainTip TracingOff _ _ _ = nullTracer
+teeTraceChainTip (TracingOn trSel) elided trTrc trMet =
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:360]: Why are we adding an additional parameter to `teeTraceChainTip` when both parameters accept the same tracer?  Can't we move `appendName` into `teeTraceChainTip`?

Comment (deepfire) [reply cardano-node/src/Cardano/Tracing/Tracers.hs:360]: @Jimbo4350, it's to make it a bit more apparent at the top level of the call chain that we're posting metrics to two different namespaces.

**Review by Jimbo4350 — APPROVED** [review]
LGTM with the caveat that devops should look at this PR wrt changes made to metrics

**Review by jbgi — APPROVED** [review]

**Pull request merged**